### ISSUE TITLE
Duplicating nodes with text is very slow

### ISSUE THREAD

**Issue by JekSun97** [godotengine/godot#93402]
### Tested versions

Godot 4.2.2 , Godot 4.3 beta2

### System information

Godot v4.2.2.stable - Windows 10.0.19045 - Vulkan (Mobile) - dedicated Radeon RX 560 Series (Advanced Micro Devices, Inc.; 31.0.14001.45012) - Intel(R) Core(TM) i5-4570 CPU @ 3.20GHz (4 Threads)

### Issue description

I noticed that duplicating Button type nodes greatly reduces the productivity of the editor, I also checked this with Label, and there were also freezes, perhaps this is due to the text.

I tested it on other types of graphic nodes, and I didn’t notice any slowdown in the editor.

### Steps to reproduce

1. Open a project
2. Duplicate the "Panel" node using Ctrl+D and quickly press it several times to make sure there are no freezes.
3. Also duplicate the Button node with a quick press to see the editor slow down

### Minimal reproduction project (MRP)

[test.zip](https://github.com/user-attachments/files/15917616/test.zip)

**Comment by aaronp64** [comment]
Looks like most of the time is spent updating the Inspector tab to reflect the new duplicated node, especially looking up theme information for the controls being added to the Inspector.  I see similar delays just changing the selected node quickly, Button is slower than Panel there too, might be based on how many (or what types of) properties need to be added to the Inspector.

**Comment by Sluggernot** [comment]
Weirdly, (I do NOT have a profiler on this) but selecting 15 Buttons and pressing Ctrl+D doesn't appear to take any longer than duplicating 1 button, including when spamming.

**Comment by JekSun97** [comment]
> Weirdly, (I do NOT have a profiler on this) but selecting 15 Buttons and pressing Ctrl+D doesn't appear to take any longer than duplicating 1 button, including when spamming.

I agree, I also don’t see a difference in performance when duplicating one button or several buttons at once, which is quite strange, observing the slowdown when duplicating one button.

**Comment by Calinou** [comment]
Closing per the above comment.
